community/mission.md

```diff
@@ -0,0 +1,22 @@
+---
+breadcrumb: Mission and Values
+---
+# Our Mission and Values
+Our mission and values serve not just as statements on a page, but as a compass, directing our work, shaping our decisions, and reminding us of the purpose that brought us together. They embody our identity, our aspirations, and the ethical path we've chosen to navigate through the complex realms of privacy and security. As you read on, know that these are not mere words, but the living, breathing essence of who we are and what we steadfastly strive for.
```

Comment:
Line seems a bit long. Break it like you have for the mission statement section below?

Reply:
I feel otherwise, the rest of the lines should be long. I overlooked that too, tbh. 
It's web/render responsibility to wrap the lines accordingly. No?

Reply:
And I've just checked with the rest .md pages, we do use wrapped lines. Adjusted in this commit.